sample_workloads/nccltest/gke/values.yaml

```diff
@@ -0,0 +1,60 @@
+cluster:
+  nNodes: null  # Must specify on commandline (--set cluster.nNodes=2)
+  sbPlacement: false # Set `true` if running across multiple superblocks
+  nSuperblocks: 1
+  startSuperblock: 1
+  gcsBucket: "pirillo-sct-bucket"
```

Comment:
Default value not accessible externally (would recommend just removing GCS dependency).